server/model/numbered-list.js

```diff
@@ -0,0 +1,6 @@
+import {Record} from 'immutable';
+
+export const NumberedList = Record({
+  heading: null,
```

Comment:
Could we use `name` or `description` here to match the other schema's (and schema.org)?